lib/rabbit_feed/event_definitions.rb

```diff
@@ -44,6 +45,7 @@ def payload_contains &block
       end
 
       def field name, options
+        @sensitive_fields << name.to_s if options.delete(:sensitive)
         fields << (Field.new name, options[:type], options[:definition])
```

Comment:
> This modifies it whilst preventing people from changing sensitive fields from outside the class.

I don't think that's the case: the `attr_reader` on both `@fields` and `@sensitive_fields` allow them to be changed outside the class already: you get a "read-only" handle on the collection, but have free play with the collection's content...